devices/ikea.js

```diff
@@ -1025,4 +1025,30 @@ module.exports = [
         description: 'PILSKOTT LED floor lamp',
         extend: tradfriExtend.light_onoff_brightness(),
     },
+    {
+        zigbeeModel: ['VINDSTYRKA'],
+        model: 'E2112',
+        vendor: 'IKEA',
+        description: 'Vindstyrka Air Quality and Humidity sensor',
+        fromZigbee: [fz.temperature, fz.humidity, fz.heiman_pm25],
```

Comment:
Is this a battery  powered device? If so, you probably want to add `ikea.fz.battery` here and also bind genPowerCfg.batteryPercentRemaining for reporting in the configure function.

Reply:
It is not battery powered